### ISSUE TITLE
how to run this on localhost?

### ISSUE THREAD

**Issue by couscousbawl** [PaystackHQ/PaystackJS-Sample-code#7]
Literally all I did is download this git and run it on my localhost, changed the secret key and opened the path of the folder on my browser!

Showing this on my browser,
Your server is set up.
/ does not exist
Open /payment to test the form. 

How do I get to test this? what is the recognized paths?

**Comment by adescode** [comment]
change your /index.php to /payment it should work. 
i'm getting 
Curl failed with response: 'SSL certificate problem: unable to get local issuer certificate'.

**Comment by ibrahimlawal** [comment]
@teabread90 apologies for the tardy response... I guess you already figured out that the recognized path is `/payment` tho.

@adescode this article should help you understand:  https://github.com/yabacon/paystack-php/wiki/cURL-error-60:-SSL-certificate-problem:-unable-to-get-local-issuer-certificate-(see-http:--curl.haxx.se-libcurl-c-libcurl-errors.html) . 

TL;DR > Your PHP server is old and needs an update.

**Comment by alagaddonjuan** [comment]
@teabread90  all you have to do is link it to the folder named paystack and index.php will direct it to pay.php or html and that would be solved

**Comment by winniegakuru** [comment]
> change your /index.php to /payment it should work.
> i'm getting
> Curl failed with response: 'SSL certificate problem: unable to get local issuer certificate'.

Still not working
